### ISSUE TITLE
Add 'share your story' box on homepage

### ISSUE THREAD

**Issue by katjam** [geeksforsocialchange/teamwilder#278]
## Acceptance Criteria

- [ ] Share your story box appears on home page
- [ ] The share your story call to action links through to the submit story page #34 
- [ ] It's not on the home page design but should look like the one in the image and sit at the bottom of the third column.

---------

Converted to further task as it involves bespoke text:
- [ ] #336

![Screenshot 2023-05-31 at 14 36 01](https://github.com/geeksforsocialchange/teamwilder/assets/46496391/f3dc460c-724e-4e7c-89b9-bd5840bc444a)

**Comment by r-ferrier** [comment]
Blocked by #34

**Comment by katjam** [comment]
Need to think about this - do we present stories on guide pages at all anymore? See #328

**Comment by katjam** [comment]
Question: The copy for this box on guide page will need to be per guide - which may cause translation issues for @AutumnBarlow .

I suggest we implement as a block on home page with sitewide / generic copy and then make a nice to have that targets the specific guide.
